packages/boxel/addon/components/boxel/modal/usage.hbs

```diff
@@ -25,7 +25,7 @@
 </Boxel::Modal>
 </:example>
 <:api as |Args|>
-  <Args.String @name="size" @description="Can be `small`, `medium`, `large`, or unspecified. Sets a predetermined max-width to inner modal." @value={{this.size}} @options={{array "small" "medium" "large" "n/a"}} @onInput={{fn (mut this.size)}} @defaultValue="65rem" />
+  <Args.String @name="size" @description="Can be `small`, `medium`, `large`, or unspecified. Sets a predetermined max-width to inner modal." @value={{this.size}} @options={{array "small" "medium" "large" undefined}} @onInput={{fn (mut this.size)}} @defaultValue="65rem" />
```

Comment:
I tried this out in Freestyle and found that it doesn’t allow `undefined` to be chosen after another value has been chosen.

Reply:
Fixed this.